README.md

```diff
@@ -129,6 +162,16 @@ additional options
   * "yes" the Makefile will always act as "IS_BUILDBOT" was set to "yes"
   * "no"  the Makefile will always act as "IS_BUILDBOT" was set to "no" / is unset. This way we can run builds on the buildbot like a local build.
 
+
+### Build a new router model for an existing architecture
+
+Everything that is in the [Table of Hardware](https://openwrt.org/toh/start) should work at least in theory. Wifi should be ath9k / ath10k. mt76 also appears to work. 
+
+You can use the Imagebuilder to get a list of all supported models for a platform. You can also look in the [source](https://git.openwrt.org/?p=openwrt/openwrt.git;a=tree;f=target/linux/ath79/image) for model names. 
+
+Just put the name in the `/profiles/<target>.profiles` and it should work. Assure, that there is also a matching configuration for that target at `/configs/`
+
```

Comment:
Unify the use of target vs. platform. In line 170 you write "all supported models for a platform", in line 172 "matching configuration for that target ".
I prefer using the target term, as it's also "upstream" speak.

Reply:
also remove the leading "/" for the folders.

Reply:
Okay, both done.